initial_bot/mentor_bot_test.py

```diff
@@ -1,16 +1,58 @@
-# bot.py
-import pdb
 import sys
-import os
 import random
 from typing import Dict
 import discord
 import argparse
+import logging
 from discord.ext import commands
 from discord.utils import get
 import test_api as api 
 
 
+url = "http://127.0.0.1:8000/api"
+
+STUDENT_ROLE= f"{url}/roles/1/"
+MENTOR_ROLE= f"{url}/roles/2/"
+SUPER_MENTOR_ROLE= f"{url}/roles/3/"
+
+def get_member_obj(id,role,channel) -> Dict :
+    return {
+        "role":role,
+        "room" : f"{url}/rooms/{channel.id}/",
+        "profile":f"{url}/profiles/{id}/",
+        }
+
+
+def create_overwrites(guild, members) :
+            overwrites = {
```

Comment:
These are indented a lot more than 4 spaces in. Are you sure you got `black` to work?